src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$calendar-cell-size: 100px;
+$calendar-padding: 10px;
+$gap-size: 1px;
+$main-color: #eee;
+$hovered-cell-color: #ffbfcb;
+$days-of-week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap-size;
+  flex-wrap: wrap;
+  width: (7 * $calendar-cell-size) + ($calendar-padding * 2) + ($gap-size * 6);
+  padding: $calendar-padding;
+
+  &__day {
+    background-color: $main-color;
+    width: $calendar-cell-size;
+    height: $calendar-cell-size;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    border: 1px solid black;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        position: absolute;
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+    }
+
+    &:hover {
+      box-sizing: border-box;
+      cursor: pointer;
+      background-color: $hovered-cell-color;
+      transform: translateY(-20px);
+      transition: all 0.5s ease-in-out;
```

Comment:
Move it to calendar__day, because this effect should be smooth in both cases: when mouse enter and when it leaves

Reply:
ok